src/main/java/core/basesyntax/Color.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Color {
+    blue,
```

Comment:
![image](https://user-images.githubusercontent.com/12864128/192154495-1827f136-1de1-4e20-8cbe-d5c259a9503f.png)
All values in enum are constant.